### PR TITLE
Feat: Move remaining utils to root dir

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/protocols/aeternity/helpers/index.ts
Comment (subhod-i) [inline src/protocols/aeternity/helpers/index.ts:1]: What do you think about renaming the methods in this file to include Aeternity in it?
For eg: `checkAddress` to `checkAeternityAddress`

**Review by subhod-i — COMMENTED** [review]
Added a comment. Rest everything looks good.

File: src/protocols/aeternity/helpers/index.ts
Comment (peronczyk) [reply src/protocols/aeternity/helpers/index.ts:1]: I'm rather in favor of this. But I didn't want to make the PR too big. Changing the names of so many methods would make more harm to the work in progress (conflicts) than actual gains. So what do you think about doing this gradually in the future?

**Review by pstachel — APPROVED** [review]

File: src/protocols/aeternity/helpers/index.ts
Comment (subhod-i) [reply src/protocols/aeternity/helpers/index.ts:1]: Fine.

**Review by subhod-i — APPROVED** [review]

**Pull request merged**